Tools/Scripts/libraries/webkitbugspy/webkitbugspy/tracker.py

```diff
@@ -55,6 +56,10 @@ def __nonzero__(self):
             return self.redacted
 
         def __repr__(self):
+            if self.exemption:
+                if self.reason:
+                    return '{} and is exempt from redaction'.format(self.reason)
+                return 'is exempt from redaction for an unknown reason'
```

Comment:
Why do we want to support the "unknown reason" case? Seems like that should be a hard error.

Reply:
One potential way to implement this suggestion is to change exemption from a boolean with a separate reason field to a combined exemptionReason field, which is either a string or null. (Or maybe use an enum instead.)

Reply:
We don't really support the "unknown reason" case, I was just being thorough in exhausting cases. I'll raise an exception in the constructor, in practice, this should be generated from `metadata/trackers.json` where we define reason in all cases.